content/features/proxy2/configure.md

```diff
@@ -0,0 +1,411 @@
++++
+title = 'Configure the built-in proxy'
+weight = 30
++++
+
+This page is about configuring and running the Memcached built-in proxy.
+
+For a general overview of the proxy, see [Proxy]({{<legacy_proxy_base_path>}}). For a hands-on tutorial that guides you through compiling and running the proxy with a simple configuration, see [Built-in Proxy Quickstart]({{<proxy_base_path>}}quickstart).
+
+## Before you begin {#prereqs}
+
+Before you and configure and run the proxy, you need the following:
+
+* Memcached, with proxy features enabled, installed on your system.
+* The proxy route library file downloaded to some path on your file system.
+
+For more information, see [Install the built-in proxy]({{<proxy_base_path>}}install).
+
+## Create a proxy configuration file {#create}
+
+Before you can run the proxy, you need to create a configuration that lists the pools of Memcached servers that you want to use as the proxy's back end, and describes the routing trees that you want the proxy to use.
+
+The configuration file is a Lua script that extends the standard proxy route library. Advanced users can modify the route library itself, or even replace it entirely with their own Lua code. The techniques described on this page focus on using the route library as-is, which covers the majority of Memcached proxy use-cases.
+
+### Overview of the configuration file
+
+The configuration file has four general parts:
+
+1. Optional invocations of global functions to turn on logging or debugging features.
+1. An optional `settings{}` block, which sets various proxy settings to values other than their defaults.
+1. A `pools{}` block, which defines groups of backend Memcached servers that the proxy routes queries to.
```

Comment:
one or more. It can actually be called multiple times (for organizational or scripting purposes, I guess)

Reply:
Done, and added a clarifying paragraph to the start of the pools{} section.